subworkflows/local/amp.nf

```diff
@@ -65,8 +74,38 @@ workflow AMP {
 
         AMP_HMMER_HMMSEARCH ( ch_in_for_amp_hmmsearch )
         ch_versions = ch_versions.mix(AMP_HMMER_HMMSEARCH.out.versions)
+        GUNZIP2 ( AMP_HMMER_HMMSEARCH.out.output )
+        ch_hmmout = GUNZIP2.out.gunzip
```

Comment:
I htink I would rather give this a more informative name, like GUNZIP_HMMER or something